packages/frontend/app/routes/peers.$peerId.tsx

```diff
@@ -121,6 +121,18 @@ export default function ViewPeerPage() {
                     defaultValue={peer.name ?? ''}
                     placeholder='Peer name'
                     error={response?.errors.general.fieldErrors.name}
+                    description={
+                      <>
+                        The name of the{' '}
+                        <a
+                          className='tooltip'
+                          href='https://rafiki.dev/concepts/interledger-protocol/peering/'
+                        >
```

Comment:
This would apply to all these links:

Should it open in a new tab? user likely wants to resume whatever they were doing here after visiting this site for more context.